websocket_server/server.js

```diff
@@ -22,53 +24,66 @@ const {
 	TLS,
 	TLS_KEY: keyPath,
 	TLS_CERT: certPath,
+	STORAGE_STRATEGY = 'redis',
```

Comment:
Also to document in .env.example

Reply:
We should probably default to LRU

```suggestion
	STORAGE_STRATEGY = 'lru',
```